pom.xml

```diff
@@ -573,6 +574,22 @@
                         <sourceDirectories>${project.build.sourceDirectory},${project.build.testSourceDirectory}</sourceDirectories>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>com.github.spotbugs</groupId>
+                    <artifactId>spotbugs-maven-plugin</artifactId>
+                    <version>${spotbugs.version}</version>
+                    <dependencies>
```

Comment:
The dependency declaration can be omitted as the spotbugs-maven-plugin and spotbugs version are the same.